tools/sst_dump_test.cc

```diff
@@ -7,6 +7,10 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file. See the AUTHORS file for names of contributors.
 
+#include <cstdint>
```

Comment:
Do we need this here?
If we do, is it possible to remove the `#include <stdint.h>` in line 16?